src/main/resources/help/content/deployment.md

```diff
@@ -100,38 +100,37 @@ Deployment notes:
 
 Download: curl -O ${source_raw_content_url_base}/${source_repo_organization}/${project_name}/master/deployment/circleci/config.yml
 
-### Deployment sample for Docker, with Detect running in container
+### Deployment sample for Docker with Detect running in a container
 
 Approach: Toolkit
 
-Deployment notes: The containers communicate via localhost URLs.
+Deployment notes: The containers communicate through localhost URLs.
 
 Download: ${source_raw_content_url_base}/${source_repo_organization}/${project_name}/master/deployment/docker/runDetectInContainer/setup.sh			
 
-### Configuring ${solution_name} for your Docker Registry
+### Configuring ${solution_name} for your Docker registry
 
-If you invoke ${solution_name} with an image reference (vs. an image that has been saved to a .tar file),
-it uses the docker-java library (${docker_java_project_url}) to access the Docker registry
+If you invoke ${solution_name} with an image reference, versus an image that has been saved to a .tar file,
+it uses the Docker-Java library (${docker_java_project_url}) to access the Docker registry
```

Comment:
docker-java is the name of the github project: https://github.com/docker-java/docker-java. I think giving them that name makes it easier to find.